hokusai/commands/config.py

```diff
@@ -0,0 +1,74 @@
+import os
+
+from collections import OrderedDict
+from subprocess import check_output, CalledProcessError, STDOUT
+import yaml
+
+from hokusai.config import HokusaiConfig
+from hokusai.common import print_red, print_green
+
+def pull_config(context):
```

Comment:
Yea [docstrings](https://www.python.org/dev/peps/pep-0257/) would be a good addition